From c9c9cbaecfcb12624bd18704945a75f60f967d37 Mon Sep 17 00:00:00 2001
From: Li Zeng <li.zeng@intel.com>
Date: Wed, 10 Apr 2013 08:20:16 +0800
Subject: [PORT FROM MAIN] psb_drm: modify psb_drm.h

BZ: 92956

modify psb_drm.h

Change-Id: I813e29803cef0025a6842ef125fc60043ba1876e
Orig-Change-Id: I5cb786a2742aae72dc50a2e7535a3a0f99826b1a
Signed-off-by: Li Zeng <li.zeng@intel.com>
Reviewed-on: http://android.intel.com:8080/100620
Reviewed-by: cactus <cactus@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Reviewed-by: Ma, Hongpeng <hongpeng.ma@intel.com>
Reviewed-by: Chen, HuaqiangX <huaqiangx.chen@intel.com>
Reviewed-by: Shi, PingX <pingx.shi@intel.com>
Tested-by: Shi, PingX <pingx.shi@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 libc/kernel/common/linux/psb_drm.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/libc/kernel/common/linux/psb_drm.h b/libc/kernel/common/linux/psb_drm.h
index 05a5813..f891309 100644
--- a/libc/kernel/common/linux/psb_drm.h
+++ b/libc/kernel/common/linux/psb_drm.h
@@ -152,6 +152,7 @@ struct psb_validate_req {
  uint32_t group;
 /* WARNING: DO NOT EDIT, AUTO-GENERATED CODE - SEE TOP FOR INSTRUCTIONS */
  uint32_t pad64;
+ uint32_t unfence_flag;
 };
 struct psb_validate_rep {
  uint64_t gpu_offset;
-- 
1.7.4.1

